types/index.d.ts

```diff
@@ -1,19 +1,20 @@
-import { Server as PretenderServer } from "pretender";
+import { RequestHandler, Server as PretenderServer } from "pretender";
 
 declare interface IServerOptions {
   environment?: string;
   routes?: (this: Server) => any;
   baseConfig?: (this: Server) => any;
   testConfig?: (this: Server) => any;
+  seeds?: (server: Server) => any;
```

Comment:
I haven't seen `seeds` in the API doc, however it is mentioned in the Guide.

API: https://miragejs.com/api/classes/server
Guide: https://miragejs.com/docs/data-layer/database#basic-usage

Reply:
You're right, we need to add this

Reply:
Thanks! I've created a PR that adds this as esdoc types. WDYT?

https://github.com/miragejs/miragejs/pull/268